prac_08/convert_miles_km.py

```diff
@@ -0,0 +1,39 @@
+"""
+Estimate time: 1hour
+Actual time: 2hours
+"""
+
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.properties import StringProperty
+
+KILOMETER_TO_MILES = 1.609344
+
+
+class ConvertMilesKmApp(App):
+    result = StringProperty()
+
+    def build(self):
+        """build the Kivy app from the kv file"""
+        self.title = "Convert Miles to KM"
+        self.root = Builder.load_file('convert_miles_km.kv')
+        return self.root
+
+    def handle_miles_to_km(self):
+        km = self.validate_miles() * 1.609344
```

Comment:
you already created the value of KILOMETER_TO_MILES in line 10 but you did not use it here